admin/package-files/flocker-control.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Flocker Control Service
+
+[Service]
+ExecStart=/usr/bin/flocker-control -p systemd:0 -a systemd:1
```

Comment:
Can we actually do TLS off of systemd-passed-sockets like this? As first approximate I would guess yes, but if not then maybe we can't use this systemd feature.